components/pagination/spec/Pagination.stories.mdx

```diff
@@ -0,0 +1,121 @@
+import { Meta, Preview, Props, Story } from '@not-govuk/docs-components';
+import { Pagination } from '../src/Pagination';
+import readMe from '../README.md';
+
+<Meta
+  title="pagination"
+  component={ Pagination }
+  parameters={ {
+    chromatic: { viewports: [640, 480] },
+    description: 'Component to allow separating long tables into separate tables',
+    jest: ['Pagination'],
+    notes: readMe
+  } }
+/>
+
+# pagination
+
+Component to allow separating long tables into separate tables with links to each
```

Comment:
Needs a full-stop. See also my point above about the description.

Reply:
updated in latest commit